class.jetpack-cli.php

```diff
@@ -944,6 +946,10 @@ public function partner_provision( $args, $named_args ) {
 			$request_body['force_connect'] = intval( $named_args['force_connect'] );
 		}
 
+		if ( 0 < $request_body['onboarding'] ) {
```

Comment:
That was because we discussed having in the future different onboarding process, partners might be able to choose one. It's a bit confusing right now since more context on the commit or in a p2 and it's not going to be implemented any time soon so I'm reverting to just check for true/false. We can always update this later.